ShopifySharp/Services/SalesChannel/CollectionListingService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// A service for interacting with Shopify product collection that a merchant has made available to your sales channel
+    /// </summary>
+    public class CollectionListingService: ShopifyService
```

Comment:
If there's no methods for this class it should be removed.

Reply:
Implemented GET methods